CODEOWNERS

```diff
@@ -6,4 +6,4 @@
 # The sequence matters: later patterns take precedence.
 
 # FILES  OWNERS
-*        @usnistgov/opensource-team
+*        @usnistgov/itl-oscal
```

Comment:
1. Do we intend for the whole team to commit or change any piece of code in any branch? (I do not ask with judgement, I am just asking because I think different repos in our presumed TBD portfolio require we decide that for this bit to work.)
2. Have you or I configured the group to have contributors permissions on the repo yet or do we need to set that up? If you are not sure, let me know and I can do it/walk you through it. :-)

Reply:
How the site is governed is tbd but I think an "or" arrangement with specific responsible parties, each of whom is trusted, would be fine (in lieu of 'entire team'). I would be willing to be on that list. Or however you wish to manage it.

Currently contributors are set to itl-oscal, me, @david-waltermire-nist and yourself.

Reply:
OK I'd say then this makes sense for now. Can you please go into [the access settings](https://github.com/usnistgov/oscal-xslt/settings/access?query=role%3Aadmin) and the following group to with the admin role: `itl-oscal-admins`? Once you do do so, I am fine with approving this PR.

Thank you for bringing this back to my attention, Wendell.

Reply:
@wendellpiez, ok I will set aside some quick time tomorrow and walk through setup with you during a screenshare.